src/NSwag.Generation.AspNetCore/NSwag.Generation.AspNetCore.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netstandard1.6;net451;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>netstandard1.6;net451;netstandard2.0;netcoreapp3.0</TargetFrameworks>
```

Comment:
Adding "netcoreapp3.0" is not an option as it would to an explosion of TFs... please use reflection as we did it for the other things... isnt System.Text.Json a .net standard 2.0 package too?

Reply:
This would not necessarily lead to an explosion of TFs. Projects targeting .net core 3.1 (and current .net 5 nightly) uses the netcoreapp3.0 target if that is the highest available in the library. I have no idea if this will be true for future versions.

System.Text.Json is .net standard, but the ASP.NET Core integration is .net core >= 3.0. AFAIK System.Text.Json support is not relevant for previous versions.  
My main reason for having the netcoreapp3.0 target was to use the #if NETCOREAPP3_0. So it's probably not required for this PR. I'll remove it and try to use reflection if that is still what you prefer.